larky/src/main/resources/vendor/luhn.star

```diff
@@ -0,0 +1,44 @@
+load("@stdlib//larky", "larky")
+
+def _sum(nums):
```

Comment:
wow, i did not know that we didn't have a sum built-in! do you mind moving this function to the[ builtins module (similar to python?)](https://docs.python.org/3/library/functions.html#sum). that way you can just do:

```python
load("@stdlib//builtins", "builtins")
builtins.sum() # ... https://docs.python.org/3/library/functions.html#sum
```

Reply:
Can do!

Reply:
Added with https://github.com/verygoodsecurity/starlarky/pull/125/commits/9d0f909365aa37e0d5a246914e390f501e499f11